examples/spawn-and-move/manifests/overlays/contracts/actions.toml

```diff
@@ -0,0 +1,2 @@
+name = "dojo_examples::actions::actions"
+reads = [ "File", "Overlay", "Test" ]
```

Comment:
This is for example I guess? As it's in example, can't this be confusing for users?
Don't we want to only override the reads to add `Moves` and `Position`?

Reply:
yeah i just added random fields to see if files are being merged properly, afaik we are not using this field anywhere at the moment so it has no effect. But yes i will update it to be `["Moves", "Position"]` so its clearer